.github/workflows/schedule-daily.yml

```diff
@@ -100,3 +104,29 @@ jobs:
           BAZEL_CI_CONFIG: "--config=ci --repository_cache=/cache/bazel"
           ZH2_DLL01_CSV_SECRETS: "${{ secrets.ZH2_DLL01_CSV_SECRETS }}"
           ZH2_FILE_SHARE_KEY: "${{ secrets.ZH2_FILE_SHARE_KEY }}"
+
+  nns-tests-nightly:
+    name: NNS Tests Nightly
+    runs-on:
+      group: dind-large
+    container:
+      image: ghcr.io/dfinity/ic-build@sha256:5bd0f059ad6e21966e9c644516b6ecd52d5ed44f1b18a76b91b59740a9d639a2
+    timeout-minutes: 180 # 3 hours
```

Comment:
upgrade_canisters_with_godlen_nns_state only takes 5 minutes on my devenv VM. If this is the only test being run here, I'd suggest reducing this to something like 2-5x (i.e. 10 - 25 min). But I guess the idea is that more tests would be added (in rs/nns with tage nns_tests_nightly) later? Currently, there are none (but ofc, this PR adds the tag to a test).

Reply:
Done.